src/store/ChartAdapterStore.ts

```diff
@@ -254,6 +254,8 @@ export default class ChartAdapterStore {
     onTouch(e: TouchEvent) {
         // Prevent vertical scroll on the chart for touch devices by forcing scroll on a scrollable parent of the chart:
         const chartNode = this.mainStore.chart.chartNode;
+        const flutterChart = document.getElementsByClassName('flutter-chart')[0] as HTMLElement;
```

Comment:
I didn't know that we have couple of elements with flutter charts className

Reply:
Is it always first one in array? If yes, we can use `querySelector` as it returns the 1st element
`document.querySelector('.flutter-chart')`

Reply:
Its one , just declaring 2 time , changed to one now